### PR TITLE
Expanded initialization steps and fixed formatting.

### PR DESCRIPTION
**Pull request by yuri**

### CODE REVIEW

**Comment by yuri** [comment]
@SethDavenport Please review retroactively.

File: handout/11-ui-router.md [hunk 1]
```diff
@@ -56,18 +56,38 @@ However the example above introduces a couple of new concepts: providers and
 `.config()` blocks.
 
 When an AngularJS application starts up, it goes throw several 'phases':
```
Comment (SethDavenport) [inline handout/11-ui-router.md]: s/throw/through/

File: handout/11-ui-router.md [hunk 2]
```diff
@@ -56,18 +56,38 @@ However the example above introduces a couple of new concepts: providers and
...
+
+6. All .run() blocks are executed in the order in which they were defined.
+
+A provider is something Angular's dependency injector can use to create a
+service. The provider can be configured with various data in the app's
+`config` phase, _before any services are instantiated_. When services are
+instantiated later, they can customize them based on configuration requests it
+received in the config phase.
+
+In practical terms, any JavaAcript object that exposes a function called $get()
```
Comment (SethDavenport) [inline handout/11-ui-router.md]: s/JavaAcript/JavaScript

**Comment by SethDavenport** [comment]
Two typos, otherwise :+1:

**Comment by yuri** [comment]
@SethDavenport Can you make a PR for those two? Note that file names have changed.

**Comment by SethDavenport** [comment]
Sure

**Comment by SethDavenport** [comment]
Done and merged.

**Pull request merged**